.travis.yml

```diff
@@ -6,8 +6,6 @@ git:
 
 jobs:
   include:
-    - python: 3.5
-      env: REQUIREMENTS=requirements-dev-3.5.txt
```

Comment:
I've just merged an existing PR that drops 3.5.

Reply:
Ah, cool. I won't be able to rebase right now (life catching up), but I will soon-ish!